README.md

```
@@ -1,39 +1,17 @@
-You can preview the production build with `npm run preview`.
-
-> To deploy your app, you may need to install an [adapter](https://svelte.dev/docs/kit/adapters) for your target environment.
--->
+# Reds
+
+Reds is a card game you can play with friends in real life. It is a game of memory, strategy and wits!  
+Our [website](https://reds-game.pages.dev) can help you learn how to play!
+I learnt it from [@ferboinkles](https://github.com/ferboinkles) and, using my technical skills, wanted to share it with you.  
+Together, we want to teach you Reds.
+
+### If you want to run it locally:
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)